### PR TITLE
Wizard recipe: kcp_jll-v1.5.0

### PR DESCRIPTION
**Pull request by hros**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: kcp_jll
* Version: v1.5.0

@staticfloat please review and merge.

### CODE REVIEW

**Comment by staticfloat** [comment]
Oooohhh, `kcp` huh?  Super useful tool last time I was behind the GFW.  :P

**Comment by staticfloat** [comment]
It looks like these build steps don't quite work; is there a reason you're not using the provided cmake scripts?

**Comment by hros** [comment]
included cmake file (CMakeLists.txt) creates only a static library
so does their `vcpkg` script (recommended in the project's README file)
I made the necessary changes to the cmake file:
add_library(kcp STATIC ikcp.c) ==> add_library(kcp SHARED ikcp.c)
ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} ==> LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
and it now creates a shared library
modified my pr

File: K/kcp_jll/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+sources = [
+    GitSource("https://github.com/skywind3000/kcp.git", "87c0e6a92bb7d8b49937522cdd5844905d58d8e6"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd kcp/
+cat CMakeLists.txt | sed 's/STATIC/SHARED/' | sed 's/ARCHIVE/LIBRARY/' > CMakeLists.txt.new
+mv CMakeLists.txt.new CMakeLists.txt
```
Comment (giordano) [inline K/kcp_jll/build_tarballs.jl]: For the record, you can do this with a single command:
```suggestion
sed -i -e 's/STATIC/SHARED/' -e sed 's/ARCHIVE/LIBRARY/' CMakeLists.txt
```

**Pull request merged**